src/arch/src/aarch64/fdt.rs

```diff
@@ -325,8 +325,8 @@ fn create_cpu_nodes(fdt: &mut Vec<u8>, vcpu_mpidr: &Vec<u64>) -> Result<()> {
     Ok(())
 }
 
-fn create_memory_node(fdt: &mut Vec<u8>, guest_mem: &GuestMemoryMmap) -> Result<()> {
-    let mem_size = guest_mem.end_addr().raw_value() - super::layout::DRAM_MEM_START;
+fn create_memory_node(fdt: &mut Vec<u8>, guest_mem: &GuestMemory) -> Result<()> {
```

Comment:
This should be `GuestMemoryMmap`. Please double check for `GuestMemory` occurrences.

Reply:
Done. Please take a look.